Extension/src/LanguageServer/extension.ts

```diff
@@ -331,6 +331,9 @@ function realActivation(): void {
     clients.ActiveClient.notifyWhenReady(() => {
         intervalTimer = global.setInterval(onInterval, 2500);
     });
+
+    // Log cold start.
+    clients.timeTelemetryCollector.setFirstFile();
```

Comment:
We only mark an openfile event as "cold start" when there is a C/C++ file open in the editor at activation time.  I didn't see any code that checked this.  I think you could check this in the timestamp collector constructor and save the state then.  Doing it here is not measuring the right thing.